loadbalancer/etcd/publisher.go

```diff
@@ -0,0 +1,92 @@
+package etcd
+
+import (
+	"github.com/coreos/go-etcd/etcd"
```

Comment:
Please add a blank line after this import statement, to separate third party from Go kit imports.